packages/react-examples/src/react-components/FluentUIComponents/Introduction.stories.mdx

```diff
@@ -4,16 +4,16 @@ import './intro.css';
 
 <Meta title="Concepts/Introduction" />
 
-<h1 class="fluent">Fluent UI React vNext</h1>
+<h1 className="fluent">Fluent UI React vNext</h1>
```

Comment:
Q: is this change documented in storybook ?

Reply:
Yup - covered by this [PR](https://github.com/storybookjs/storybook/pull/12878)